MIGRATING.md

```diff
@@ -10,7 +10,7 @@ We have also provided a new migration tool to help with the most essential chang
 
 Our [godoc page](https://pkg.go.dev/github.com/DataDog/dd-trace-go/v2/ddtrace) should be helpful during this process. We also have the [official documentation](https://docs.datadoghq.com/tracing/setup/go/), which contains a couple of examples.
 
-This document will further outline some _before_ and _after_ examples.
+Please follow this guide for migrating from v1 to v2 and creating new services using v2 directly. This document will further outline some _before_ and _after_ examples.
```

Comment:
Adding a link would be nice.

Reply:
@kakkoyun It's referring to the guide itself. Where should we link?